lib/ruby_terraform.rb

```diff
@@ -9,6 +10,23 @@ def configuration
       @configuration ||= Configuration.new
     end
 
+    def logger(custom_path = "")
```

Comment:
I think I'd prefer if the configuration of file logging was performed by the consumer and the logger injected by the consumer rather than having all these defaults here. This provides a lot more flexibility and keeps this simpler.

Would you be able to simplify a bit and make the default logger log only to `$stdout` at info level?

That way if someone had a specific log configuration they could say:
```ruby
RubyTerraform.configure do |configuration|
  configuration.logger = Logger.new(MultiIO.new(STDOUT, File.open('my_log_file.log', 'a')), level: :debug)
end
```

This will require that `RubyTerraform::Configuration` additionally support setting a logger.